build.gradle.kts

```diff
@@ -629,9 +630,17 @@ tasks {
     destinationDirectory.set(projectDir.dir("distribution"))
   }
 
+  val assembleSparkConnector by register("assembleSparkConnector", Copy::class) {
+    dependsOn("spark-connector:spark-connector-runtime:shadowJar")
+    group = "gravitino distribution"
+    finalizedBy("checksumSparkConnector")
+    from("spark-connector/spark-connector-runtime/build/libs/${rootProject.name}-spark-connector-runtime-${sparkMajorVersion}_$scalaVersion-$version.jar")
+    into(projectDir.dir("distribution"))
+  }
```

Comment:
I don't think we need this. Trino is special because we don't assemble all the jars together, so we need a command to package the jars and zip them together. But for Spark, it is more like a client side uber jar, we can simple use `--jars` or `--packages`, no need to assemble it and put into the distribution.

Reply:
ok